src/app/components/home/home.component.ts

```diff
@@ -42,7 +42,7 @@ class HomeComponent extends PageComponent implements OnInit {
   public constructor(
     private regionApi: ShallowRegionsService,
     private projectApi: ProjectsService,
-    private securityApi: SecurityService,
+    private state: BawApiStateService,
```

Comment:
yeah state is as about a general term as exists. Let's choose a better name for these variables.

`session` or `sessionState`